src/touch/gestures_bitboxbase.c

```diff
@@ -0,0 +1,385 @@
+// Copyright 2019 Shift Cryptosecurity AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#ifndef TESTING
+#include "qtouch.h"
+#include <driver_init.h>
+#else
+#include "mock_qtouch.h"
+#endif
+
+#include "gestures.h"
+#if PLATFORM_BITBOXBASE == 1
```

Comment:
is this redundant? hopefully this file would only be compiled for bitboxbase only.